docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+      // highlight-end
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+### Starting the camera
+
+Jellyfish Client provides a handy hook for managing the camera: `useCamera`. It
+can not only start a camera but also toggle, switch between multiple cameras,
+manage camera state and camera track simulcast settings and bandwidth. Also when
+starting the camera you can provide multiple different settings such as
+resolution, quality, and metadata. In this example though we'll simply turn it
+on to stream the camera to the dashboard with default settings:
+
+```tsx title="/screens/Connect.tsx"
+import {
+  useJellyfishClient,
+  // highlight-next-line
+  useCamera,
+} from "@jellyfish-dev/react-native-client-sdk";
+
+// ...
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  // ...
+
+  // highlight-next-line
+  const { startCamera } = useCamera();
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+
+      // highlight-next-line
+      await startCamera();
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+### Joining the room
+
+The last step of connecting to the room would be actually joining the room - so
+that your camera track is visible to other users. To do this simply use the `join` function
+from the `useJellyfishClient` hook.
+
+You can also provide some user metadata when joining. You can put there whatever
+you want, it depends on your business logic. In our example, we provide a user name.
+
+After joining the room we navigate to the next screen: Room screen.
+
+```tsx title="/screens/Connect.tsx"
+// ...
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  // highlight-next-line
+  const { connect, join } = useJellyfishClient();
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+
+      await startCamera();
+
+      // highlight-next-line
+      await join({ name: "Mobile RN Client" });
+      // highlight-next-line
+      navigation.navigate("Room");
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+Now the app is ready for the first test. If everything went well you should see
+a video from your camera in the front-end dashboard. Now onto the second part:
+displaying the streams from other participants.
+
+## Displaying streams from other participants
+
+### Displaying video tracks
+
+The Room screen displays the video from the camera on your device and also
+videos from other participants as well. It also allows the user to exit the call.
```

Comment:
The Room screen has a couple of responsibilities:
* it displays your own video. Note that it's taken directly from your camera i.e. we don't send it to the JF and get it back so other participants might see you a little bit differently 
* it presents current room state so participants list, their video tiles, etc.
* it allows you to leave a meeting